File: QuevedoVirtualClassrooms-Server/src/main/java/com/quevedo/virtualclassroomsserver/common/models/dto/resource/ResourceLiteGetDTO.java
package com.quevedo.virtualclassroomsserver.common.models.dto.resource;

import lombok.Builder;
import lombok.Data;

import com.quevedo.virtualclassroomsserver.common.models.common.ResourceType;
import java.time.LocalDateTime;

@Data
@Builder
public class ResourceLiteGetDTO {
    private String uuidResource;
    private String resourceName;
    private String resourceUrl;
    private LocalDateTime timestamp;
    private ResourceType resourceType;
}

File: QuevedoVirtualClassrooms-Android/app/src/main/java/org/quevedo/quevedovirtualclassrooms/data/models/resource/ResourceLiteGetDTO.kt
package org.quevedo.quevedovirtualclassrooms.data.models.resource

import org.quevedo.quevedovirtualclassrooms.data.models.common.ResourceType
import java.time.LocalDateTime

data class ResourceLiteGetDTO(
    val uuidResource: String,
    val resourceName: String,
    val resourceUrl: String,
    val timestamp: LocalDateTime,
    val resourceType: ResourceType
)
